src/displayBackend/DisplayCommandHandler.cpp

```diff
@@ -104,6 +104,7 @@ DisplayCommandHandler::~DisplayCommandHandler()
 	LOG(mLog, DEBUG) << "Delete command handler, connector name: "
 					 << mConnector->getName();
 
+	mConnector->stopProcessing();
```

Comment:
1. Commit message: add space `callbacks (`
2. I believe such an issue should be resolved by deleting/creating object order: I don't see someone else owning the connector. If it is the case, stop processing functionality can be called in connector destructor.

Reply:
Done